### PR TITLE
Add fixture test, validate minimum Gradle version

### PR DESCRIPTION
**Pull request by JakeWharton**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,125 +1,11 @@
...
-.idea/**/markdown-navigator.xml
-.idea/**/markdown-navigator/
-
-### Gradle ###
 .gradle
 build/
-!**/src/**/build/
-
-# Local configuration file (sdk path, etc)
-local.properties
```
[the remark this answers is not in the document]
Comment (drewhamilton) [reply .gitignore]: (`PokoSample:compose` is an Android project)

**Comment by drewhamilton** [comment]
Will merge as-is for now so we at least have something in place, and then we can improve from there.
